docs/SPIR-V.rst

```diff
@@ -713,6 +716,21 @@ We will have the following offsets for each member:
 ``g_float2_2``   160    160    176    112    88    128
 ============== ====== ====== ====== ====== ====== ======
 
+If the above layout rules do not satisfy your needs and you want to manually
+control the layout of struct members, you can use either
+
+* The native HLSL ``:packoffset()`` attribute: only available for cbuffers; or
+* The Vulkan-specific `[[vk::offset()]]`` attribute: applies to all resources.
```

Comment:
missing a single \` at the beginning of `[[vk::offset()]]``

Reply:
Good catch! Done!